target/linux/mpc85xx/Makefile

```diff
@@ -11,7 +11,7 @@ BOARD:=mpc85xx
 BOARDNAME:=Freescale MPC85xx
 CPU_TYPE:=8540
 FEATURES:=squashfs ramdisk
-SUBTARGETS:=p1010 p1020 p2020
+SUBTARGETS:=p1010 p1020 p2020 p2041
```

Comment:
Is this really necessary? Having one subtarget per device is not really an option.

Reply:
No, sorry. I will try fitting this under the p2020 subtarget. The [Wikipedia page for QorIQ](https://en.wikipedia.org/wiki/QorIQ) lists out how the processors differ, but as far as OpenWrt cares I think those differences can be encoded per-board -- this was mostly copy-pasted from 5de6aed4.